ui-admin/src/study/participants/enrolleeView/ParticipantNoteView.tsx

```diff
@@ -32,42 +44,52 @@ export function ParticipantNoteView({
   const matchedTask = linkedTasks.find(task => task.participantNoteId === note.id)
   const [showTaskEdit, setShowTaskEdit] = useState(false)
   const matchedAssignedUser = users.find(user => user.id === matchedTask?.assignedAdminUserId)
-  return <div className="mb-3">
-    <div className="d-flex align-items-center">
-      <div className="fw-bold text-muted">
-        <FontAwesomeIcon icon={faUser} className="mx-2"/>
-        {usernameString}
+  const isComplete = matchedTask?.status === 'COMPLETE'
+  return <div className="mb-2">
+    <div className="d-flex align-items-start">
+      <div>
+        <div className="fw-bold text-muted">
+          <FontAwesomeIcon icon={faUser} className="me-2"/>
+          {usernameString}
+        </div>
+        <div className="text-muted">
+          <FontAwesomeIcon icon={faClock} className="me-2"/>
+          {instantToDefaultString(note.createdAt)}
+        </div>
       </div>
-      <div className="text-muted ms-3">{instantToDefaultString(note.createdAt)}</div>
-      { matchedTask && <button className="badge ms-auto btn-secondary"
-        style={{ backgroundColor: '#eee' }}
-        onClick={() => setShowTaskEdit(!showTaskEdit)}>
+      { matchedTask && <button className={classNames(
+        isComplete ? 'badge bg-success' : 'badge bg-secondary',
+        'ms-auto border-0 p-2'
+      )}
+      onClick={() => setShowTaskEdit(!showTaskEdit)}>
         { matchedTask.status === 'NEW' && <span>
+          <FontAwesomeIcon icon={faEmptyCircle} className="me-2"/>
           { matchedAssignedUser ?
               `Assigned: ${matchedAssignedUser.username}` : 'Unassigned' }
-          <FontAwesomeIcon icon={faCaretDown} className="fa-sm ms-2"/>
         </span> }
-        { matchedTask.status === 'COMPLETE' && <span>
-          <FontAwesomeIcon icon={faCheck} className="fa-sm me-2"/>
+        { matchedTask.status === 'IN_PROGRESS' && <span>
+          <FontAwesomeIcon icon={faCircleHalfStroke} className="me-2"/>
+          { matchedAssignedUser ?
+              `Assigned: ${matchedAssignedUser.username}` : 'Unassigned' }
+        </span> }
+        { isComplete && <span>
+          <FontAwesomeIcon icon={faCircleCheck} className="me-2"/>
             Resolved: {instantToDateString(matchedTask.completedAt)}
         </span> }
-
       </button>
```

Comment:
while we're cleaning this up: the entire time I've been at juniper I had no clue these were clickable and editable (lol), maybe if we put an edit icon here it would be clearer? Like just a https://fontawesome.com/icons/pen-to-square?f=classic&s=solid

Reply:
nice, I like that idea. done!